### PR TITLE
chore: add docs for terraform provider

### PR DESCRIPTION
**Pull request by ecmadao**
图片

### CODE REVIEW

File: content/docs/get-started/work-with-terraform/overview.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    bytebase = {
+      source = "bytebase/bytebase"
+      version = "0.0.2"
+    }
+  }
+}
+
+provider "bytebase" {
+  email        = "<Your Bytebase account email>"
+  password     = "<Your Bytebase account password>"
```
Comment (tianzhou) [inline content/docs/get-started/work-with-terraform/overview.md:43]: I feel this is OK for testing purpose, but in prod, we likely need to introduce service account / API key

File: content/docs/get-started/work-with-terraform/overview.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  bytebase_url = "<Your Bytebase OpenAPI URL>"
```
Comment (tianzhou) [inline content/docs/get-started/work-with-terraform/overview.md:44]: What's the common practice for tf provider here, should we provide the full URL, or just provide the bytebase external URL and let our code derive the exact path?

Comment (ecmadao) [reply content/docs/get-started/work-with-terraform/overview.md]: Make sense, I updated the wording here.

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/docs/get-started/work-with-terraform/overview.md (hunk 1 above)
Comment (tianzhou) [reply content/docs/get-started/work-with-terraform/overview.md:43]: We can change this in the followup PR

**Pull request merged**